solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -279,4 +277,8 @@ export default {
 a.search_resource_btn:visited{
     color:white
 }
+a.search_resource_btn:hover{
+    color:white;
+    background-color: #046791;
```

Comment:
background-color needs to be removed - `.default-btn:hover` takes care of the hover color, which is `$mid_dark_blue`

Reply:
updated